### ISSUE TITLE
/root/games/video --- mods ?

### ISSUE THREAD

**Issue by ashtonx** [roboyoshi/datacurator-filetree#81]
I noticed no structure for game mods... computer/os/game might not work as many mods don't really care what you play it on.
For console it's easier to keep them in same folder as game title...

**Comment by Collisteru** [comment]
You could have a game folder in which you place both the game itself and all the mods for it.

**Comment by roboyoshi** [comment]
Yeah I think that's how I did it too. Have a folder for the Game and Put everything into that folder. I think I had something like `/games/microsoft-windows/<game>/{artwork,discs,installer,mods,...}.
